jwst/pipeline/calwebb_image2.py

```diff
@@ -1,21 +1,24 @@
 #!/usr/bin/env python
-from ..stpipe import Pipeline
+from collections import defaultdict
+import logging
+
 from .. import datamodels
-import os
+from ..associations.load_as_asn import LoadAsLevel2Asn
+from ..stpipe import Pipeline
 
 # calwebb IMAGE2 step imports
 from ..assign_wcs import assign_wcs_step
 from ..flatfield import flat_field_step
 from ..photom import photom_step
 
 
-__version__ = "2.2"
+__version__ = "2.3"
```

Comment:
I think this is a sufficiently major revision (supporting ASN) to deserve a full version increment, i.e. 3.0